scripts/devenv.py

```diff
@@ -89,6 +89,9 @@ def remove_package(self, package: str, *, force=False) -> None:
         """Remove a package in this environment."""
         run("conda", "remove", "--yes", "--force" if force else None, "--name", self.name, package)
 
+    def develop(self, path: pathlib.Path):
```

Comment:
```suggestion
    def develop(self, path: pathlib.Path) -> None:
    """Install a package in development mode."""
```

Also, maybe rename to `install_package_dev` or something similar?